### PR TITLE
Updates to CSV data downloads

### PR DESCRIPTION
**Pull request by paninee**

### CODE REVIEW

File: api/helpers/create-csv-data-dump.js [hunk 1]
```diff
@@ -191,8 +248,9 @@ function convertToIdTitleUrlFields(entry, field) {
 
 async function createCSVDataDump(type, results = []) {
   var entries = results;
+  var csvFields = Object.create({});
 
-  if (type !== 'thing') {
+  if (type === 'thing') {
```
Comment (ascott) [inline api/helpers/create-csv-data-dump.js:253]: @paninee what is this change doing?

**Comment by ascott** [comment]
@pan i just pulled this down to take a look and i'm seeing some errors. are you seeing these as well? 

Screen Shot 2020-10-28 at 12 17 00 PM
Screen Shot 2020-10-28 at 12 19 01 PM

**Comment by paninee** [comment]
@ascott I can't reproduce the issue you're having. Is it possible you have bad data in your database?

**Comment by ascott** [comment]
@paninee are you going to open this PR again? would love to get it in before you go away.

**Pull request closed without merging**